environments/core/fruition

```diff
@@ -0,0 +1 @@
+/Users/david.paul/Desktop/fruition
```

Comment:
This should be removed

Reply:
oops... i need to watch this i think i do that every time